sys/pylon/gstpylonsrc.c

```diff
@@ -3050,6 +3065,43 @@ gst_pylonsrc_set_bandwidthReserveAcc (GstPylonSrc * src)
   return FALSE;
 }
 
+static gboolean
+gst_pylonsrc_set_maxTransferSize(GstPylonSrc *src)
+{
```

Comment:
Brackets should be on the same line with function name. Please run `gst-indent` on your changes

Reply:
what `gst-indent` do you use?
the one in https://github.com/GStreamer/common/blob/master/gst-indent is missing the `--braces-on-func-def-line` option to enforce your format requirement ...

Reply:
just checking core gstreamer code... it is always brace on next line
so this would be the option  `--braces-after-func-def-line`

e.g.: https://github.com/GStreamer/gstreamer/blob/b4ca58df7624b005a33e182a511904d7cceea890/gst/gstparse.c#L158

Reply:
Sorry,  my bad. I meant braces after `if`

Reply:
did some more tiny fixes, so that we can now have the gstreamer pre-commit hook in place, to enforce the commit rules